packages/nameguard-sdk/src/index.ts

```diff
@@ -372,27 +376,50 @@ export interface ConsolidatedNameGuardReport extends ConsolidatedReport {
 /**
  * NameGuard report that contains the full results of all `checks` on all `labels` in a name.
  */
-export interface NameGuardReport extends ConsolidatedNameGuardReport {
+export interface AbstractNameGuardReport extends ConsolidatedNameGuardReport {
+  /* Whether or not the name was inspected. */
+  inspected: boolean;
+
   /* The results of all checks performed by NameGuard on `name`. */
   checks: CheckResult[];
 
-  /** Details of the inspection of all labels in `name`. */
-  labels: LabelGuardReport[];
+  /** Details of the inspection of all labels in `name`.
+    *
+    * `undefined` if `name` was uninspected and this is an `UninspectedNameGuardReport`.
+  */
+  labels?: LabelGuardReport[];
 
   /**
    * The name considered to be the canonical form of the analyzed `name`.
    *
-   * `null` if and only if the canonical form of `name` is considered to be undefined.
+   * `undefined` if and only if the canonical form of `name` is considered to be undefined or `name` was uninspected and this is an ``UninspectedNameGuardReport`.
    *
    * If a label is represented as `[labelhash]` in `name`,
    * the `canonical_name` will also contain the label represented as `[labelhash]`.
    *
    * `canonical_name` is guaranteed to be normalized with the exception of the case
    * where `normalization` is `unknown` and one or more labels are represented as `[labelhash]`.
    */
-  canonical_name: string | null;
+  canonical_name?: string;
```

Comment:
@notrab Hey, this field now might be undefined rather than null. Can you please investigate and confirm if we need to update anything in `nameguard-react`, `examples.nameguard.io` or `nameguard.io`?